src/components/MoneyRequestConfirmationList.js

```diff
@@ -247,7 +247,9 @@ function MoneyRequestConfirmationList(props) {
      */
     const navigateToReportOrUserDetail = (option) => {
         if (option.accountID) {
-            Navigation.navigate(ROUTES.getProfileRoute(option.accountID));
+            const activeRoute = Navigation.getActiveRoute().replace(/\?.*/, '');
+
```

Comment:
NAB: Extra blank line

Reply:
Thanks! Should i remove the line and push it to this branch?

Reply:
Nah, it's ok, it's not really significant enough to warrant follow up changes - just general feedback on newline usage 😊

Reply:
Ok! I'll do better next time